Release v12 (#107)
RobotecAI/RobotecGPULidar: README.md

    1. You may be asked to create Nvidia account to download
3. Export environment variable:
   1. `export OptiX_INSTALL_DIR=<your-OptiX-path>`.
4. Run `./setup.py --install-deps` to install RGL dependencies.
   - It will install dependencies from `apt` and [vcpkg](https://vcpkg.io/en/index.html).
5. Use `setup.py` script to build.
   - It will use CMake to generate files for build system (make) and build.
   - You can pass optional CMake and make parameters, e.g.
     - `./setup.py --cmake="-DCMAKE_BUILD_TYPE=Debug" --make="-j 16"`
   - See `./setup.py --help` for usage information.

## Building on Windows

1. Install [Microsoft Visual Studio](https://visualstudio.microsoft.com/pl/downloads/) (Visual Studio 2019 when using ROS2 extension) with **C++ CMake tools for Windows** component.
2. Install [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads) **11.4.4+**.
3. Download [NVidia OptiX](https://developer.nvidia.com/designworks/optix/downloads/legacy) **7.2**.

>    - use the default location or set environment variable `OptiX_INSTALL_DIR`

Author: ```suggestion
   - install the framework and set environment variable `OptiX_INSTALL_DIR`
```
Environment variable is not optional since we are checking it in the `setup.py` script.
